README.md

```diff
@@ -365,6 +365,128 @@ public function createVehicleAction() {
 }
 ```
 
+## Approach C: Multi-entities
```

Comment:
This is just approach B with several entities. I don't think that it needs to be documented like this.